corehq/apps/hqadmin/tasks.py

```diff
@@ -25,9 +30,22 @@ def check_pillows_for_rewind():
                 }
             )
 
+
 @periodic_task(run_every=crontab(hour=0, minute=0), queue='background_queue')
 def create_historical_checkpoints():
     today = date.today()
     thirty_days_ago = today - timedelta(days=30)
     HistoricalPillowCheckpoint.create_pillow_checkpoint_snapshots()
     HistoricalPillowCheckpoint.objects.filter(date_updated__lt=thirty_days_ago).delete()
+
+
+@periodic_task(run_every=crontab(minute=0), queue='background_queue')
```

Comment:
This might be more frequently than needed (we'll get an email every hour until that user is removed), but we can always change it if it becomes annoying.  Have you checked the live environments to see if there are currently non-dimagi users?

Reply:
I'm ok with (maybe even pro)  annoying people for a potential security breach